tools/detect_targets.py

```diff
@@ -42,24 +42,24 @@
         # Parse Options
         parser = get_default_options_parser()
```

Comment:
This call is pulling in the following unneeded options: `-m`, `-t`, `-c <--clean`, and `-o <--options>`. Perhaps this should just be a new instantiation of argparse?